src/images/maps/geom_adm0.R

```diff
@@ -1,22 +1,37 @@
 box::use(gg = ggplot2)
+box::use(dplyr)
+box::use(purrr)
+box::use(sf)
+box::use(units)
 
 box::use(../../utils/get_iso3_sf)
 box::use(../../utils/iso3_shift_longitude)
 
+
 #' Geom for country boundaries
 #'
 #' Adds country boundaries for ISO3 code. Since `get_adm0_sf()` looks for custom
 #' basemaps, then OCHA CODs, then UN Geodata services by filtering, we throw
 #' an error if the returned data is `NULL` or a 0 row data frame.
 #'
 #' @param iso3 ISO3 code
+#' @param ... Additional sf class objects with geometry
 #'
 #' @returns Geom for the country boundaries
 #'
 #' @export
-geom_adm0 <- function(iso3) {
+geom_adm0 <- function(iso3, ...) {
   sf_adm0 <- get_iso3_sf$get_iso3_sf(iso3, "adm0")
 
+
+  additional_geoms <- list(...)
+  if (length(additional_geoms) > 0) {
+    purrr$map(
```

Comment:
`purrr$walk` because we don't want return values, this is for the side effect of assertion.

Reply:
true

Reply:
implemented